bin/mapbox-gl-style-build

```diff
@@ -68,6 +79,17 @@ const buildStyles = async (styleDir, layerDir, outputDir, options = {}) => {
       process.exit(1);
     }
   });
+
+  const neverUsedVars = [...unusedContext].filter(
+    path => !usedContext.has(path)
+  );
+
+  neverUsedVars.forEach(unusedVar => {
+    console.warn(`  Unused context variable ${chalk.blue(unusedVar)} found:`);
+    console.warn(`    This variable is not used in any style variant.`);
+  });
```

Comment:
I wonder if we could say something like "Style built with the following warnings:" then log these out? When I build a style that has unused variables the warning makes me think it was possibly unsuccessful:

```bash
$ bin/mapbox-gl-style-build --style-dir=examples/unused-context/templates/styles --layer-dir=examples/unused-context/templates/layers --out-dir=examples/unused-context/build
  Unused context variable colors.backgroundHigh found:
    This variable is not used in any style variant.

```